docs/user-guide/cli.md

```diff
@@ -38,6 +38,12 @@ stylelint "foo/**/*.scss" --syntax scss
 
 The above can be slightly altered to read Less or SugarSS syntax: `--syntax less`, `--syntax sugarss`.
 
+```shell
+stylelint "foo/**/*.html" --extract
+```
+
+Extract `<style>` tags from html and lint.
```

Comment:
In the other examples, the description is above the code sample and ends with a colon.